Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "odht"
 version = "0.1.0"
-authors = ["Michael Woerister <michaelwoerister@posteo.de>"]
+authors = ["The Rust Project Developers", "Michael Woerister <michaelwoerister@posteo.de>"]
```

Comment:
This field is no longer necessary.

Reply:
Can you explain? I just copied that from https://github.com/rust-lang/rustc-hash/blob/master/Cargo.toml.

Reply:
See [RFC 3052](https://github.com/rust-lang/rfcs/blob/master/text/3052-optional-authors-field.md).

> The crates.io registry does not allow users to change the contents of already published versions: this is highly desirable to ensure working builds don't break in the future, but it also has the unfortunate side-effect of preventing people from updating the list of crate authors defined in Cargo.toml's package.authors field.
>
> This is especially problematic when people change their name or want to remove their name from the Internet, and the crates.io team doesn't have any way to address that at the moment except for deleting the affected crates or versions altogether. We don't do that lightly, but there were a few cases where we were forced to do so.
>
> The contents of the field also tend to scale poorly as the size of a project grows, with projects either making the field useless by just stating "The $PROJECT developers" or only naming the original authors without mentioning other major contributors.

Reply:
Good to know. Thanks for the pointer, @bjorn3!